prefs.js

```diff
@@ -156,6 +161,142 @@ function buildPrefsWidget() {
     Gio.SettingsBindFlags.DEFAULT
   );
 
+  // Custom CSS stylesheet
+
+  if (ShellVersion >= 42) {
+    var custom_css_box = new Gtk.Box({
+      orientation: Gtk.Orientation.HORIZONTAL,
+      css_classes: Array("linked"), // Style class in libadwaita
+      halign: Gtk.Align.END,
+      valign: Gtk.Align.CENTER,
+    });
+  } else {
+    var custom_css_box = new Gtk.Box({
+      orientation: Gtk.Orientation.HORIZONTAL,
+      spacing: 5,
+      halign: Gtk.Align.END,
+      valign: Gtk.Align.CENTER,
+    });
+  }
+
+  let custom_css_label = new Gtk.Label({
+    label: 
+      "Custom CSS stylesheet\r" +
+      "<small>You need to reload this extension to see the changes.</small>",
+    halign: Gtk.Align.START,
+    use_markup: true,
+  });
+
+  if (ShellVersion < 40) {
+    let custom_css_help_image = new Gtk.Image({
+      icon_name: "dialog-information-symbolic",
+    });
+
+    var custom_css_help = new Gtk.Button({
+      image: custom_css_help_image,
+      tooltip_text: "Click for more information",
+      halign: Gtk.Align.END,
+      valign: Gtk.Align.CENTER,
+    });
+  } else {
+    var custom_css_help = new Gtk.Button({
+      icon_name: "dialog-information-symbolic",
+      tooltip_text: "Click for more information",
+      halign: Gtk.Align.END,
+      valign: Gtk.Align.CENTER,
+    });
+  }
+
+  custom_css_help.connect('clicked', () => {
+    Gio.Subprocess.new(
+      Array(
+        "xdg-open",
```

Comment:
As far as I know, using `xdg-open` to open websites is the best approach, since it is part of the Freedesktop specification.

> I'm not sure this works in wayland

Works on my machine (GNOME Wayland 43.2)